pom.xml

```diff
@@ -320,6 +311,27 @@ under the License.
         </exclusion>
       </exclusions>
     </dependency>
+    <!-- VerboseGraphBuilder & VerboseSerializer -->
```

Comment:
I wouldn't include this comment since these are not the artifacts where those classes are found.